.editorconfig

```diff
@@ -0,0 +1,3 @@
+# C# files
```

Comment:
Can you change this in it's own PR, it's pretty much ruined the diff for this PR.

Reply:
OK let me try, because before adding the file I also got tons of diffs when I ran `dotnet run format-sdk`

Reply:
That did not help at all 😞 now more diffs. Maybe `dotnet format` is not the same across machines _by default_ and it does need config?

Reply:
Yes do the format change in another PR, I'll approve that then rebase this